SECURITY_POLICY.md

```diff
@@ -0,0 +1,62 @@
+# Security
+
+The Galaxy project is strongly committed to security and responsible disclosure. We have adopted and published a set of policies specifying how we will act in response to reported security issues, in order to ensure timely updates are made available to all affected parties.
+
+## Reporting Security Issues
+
+If you believe you have discovered a security issue, please email [galaxy-committers@lists.galaxyproject.org](galaxy-committers@lists.galaxyproject.org). We ask that you not disclose the issues on the public issue tracker. Someone on that list will acknowledge your email within 2 US business days.
```

Comment:
Maybe the email should have some kind of `[SECURITY]` prefix in the subject? Given we are committing to a response time it should be clear to what emails this applies.